classes/reports/realtime/cli.php

```diff
@@ -91,7 +91,7 @@ public function __construct()
 
         $runnerResultField = new runner\result\cli();
         $runnerResultField
-            ->setSuccessColorizer(new colorizer('0;37', '42'))
+            ->setSuccessColorizer(new colorizer('0;97', '42'))
```

Comment:
This color code is out of the 8-bit color palette bounds. I'm afraid it won't work on some terminal emulators or environment (CI for example).